build/cekit/modules/builder-infinispan-operator/install

```diff
@@ -4,35 +4,26 @@ set -x -e
 
 #INSTALL_PKGS="go-toolset"
 OPERATOR_PATH=/usr/local/bin/infinispan-operator
+OPERATOR_HOME=${HOME}/infinispan-operator
+BUILD_PATH="./cmd/manager"
+GO111MODULE=on
 
 # yum -y install ${INSTALL_PKGS}
 
-REPO=infinispan-operator
-GHORG=github.com/infinispan
-GHFULL=${GHORG}/${REPO}
-GOPATH=$(go env GOPATH)
 go version
 
-mkdir -p ${GOPATH}/src/${GHFULL}
-tar xfz /tmp/artifacts/operator.tar.gz --strip-components=1 -C ${GOPATH}/src/${GHFULL}
-cd ${GOPATH}/src/${GHFULL} && pwd
+mkdir -p ${OPERATOR_HOME}
+tar xfz /tmp/artifacts/operator.tar.gz --strip-components=1 -C ${OPERATOR_HOME}
+cd ${OPERATOR_HOME} && pwd
 
-go generate ./...
-dep ensure -vendor-only
-#go vet ./...
-#go test ./...
-CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -v -a -o ${OPERATOR_PATH} github.com/infinispan/infinispan-operator/cmd/manager
+go mod vendor
+CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -o ${OPERATOR_PATH} ${BUILD_PATH}
 if [[ ! -f ${OPERATOR_PATH} ]] ; then
     echo "${OPERATOR_PATH} does not exist, aborting."
     exit 1
 fi
 
-#mv build/_output/bin/* /usr/local/bin/
```

Comment:
@rigazilla do you know, should we keep all commented lines or not?

Reply:
@dmvolod I think we can cleanup, thanks!

Reply:
Thanks for response, @rigazilla . 
Looks like ${INSTALL_PKGS} and go-toolset are RHEL specific.

Reply:
@dmvolod it's ok to remove comments by me. Let's stay with Fedora upstream

Reply:
@rigazilla , fixed, please review

Reply:
@dmvolod looks great to me! please squash the commits, I'll merge then. Thanks!

Reply:
Thanks, @rigazilla , done.